.opencode/command/speckit.taskstoissues.md

```
@@ -0,0 +1,31 @@
+1. Get the Git remote by running:
+
+```bash
+git config --get remote.origin.url
+```
+
+### Only Proceed to Next Steps if the Remote is a GitHub URL
+
+1. For each task in the list, use the GitHub MCP server to create a new issue in the repository that is
+   representative of the Git remote.
+
+**UNDER NO CIRCUMSTANCES EVER CREATE ISSUES IN REPOSITORIES THAT DO NOT MATCH THE REMOTE URL**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)